pkg/cloud/services/s3/s3.go

```diff
@@ -230,6 +235,39 @@ func (s *Service) ensureBucketPolicy(bucketName string) error {
 	return nil
 }
 
+func (s *Service) tagBucket(bucketName string) error {
+	taggingInput := &s3.PutBucketTaggingInput{
+		Bucket: aws.String(bucketName),
+		Tagging: &s3.Tagging{
+			TagSet: nil,
+		},
+	}
+
+	tags := infrav1.Build(infrav1.BuildParams{
+		ClusterName: s.scope.Name(),
+		Lifecycle:   infrav1.ResourceLifecycleOwned,
+		Name:        nil,
+		Role:        aws.String("node"),
```

Comment:
Is this a common role?

Reply:
Maybe I misunderstand the question. Do you mean to rather use `"common"` (`CommonRoleTagValue`) as we do for VPC, gateways and such? I used `"node"` because the bucket seems only used for bootstrapping purposes and therefore relates to nodes, similar to how ASGs are tagged, for example.